ci/do_ci.sh

```diff
@@ -146,12 +171,9 @@ elif [[ "$1" == "bazel.dev" ]]; then
   # This doesn't go into CI but is available for developer convenience.
   echo "bazel fastbuild build with tests..."
   echo "Building..."
-  bazel build ${BAZEL_BUILD_OPTIONS} -c fastbuild //source/exe:envoy-static
-  # Copy the envoy-static binary somewhere that we can access outside of the
-  # container for developers.
-  cp -f \
-    "${ENVOY_SRCDIR}"/bazel-bin/source/exe/envoy-static \
-    "${ENVOY_DELIVERY_DIR}"/envoy-fastbuild
+  bazel_binary_build fastbuild
+  bazel_binary_build fastbuild
```

Comment:
Do you mean this to be duplicated?

Reply:
Nope, that was a mistake from resolving conflicts.